example/lib/eip712Signing.dart

```diff
@@ -120,19 +117,17 @@ class _MyHomePageState extends State<MyHomePage> {
 
     print(typedData);
 
-    //Signing message
     String signature = await promiseToFuture(ethereum.request(RequestParams(
         method: 'eth_signTypedData_v3', params: [selectedAddress, typedData])));
     print(signature);
-    String r = "0x" + signature.substring(0, 66);
+    String r = signature.substring(0, 66);
```

Comment:
This change doesn't break anything?

Reply:
Nope. 0x wasn't required as signature begins with 0x